ibis/backends/tests/test_io.py

```diff
@@ -428,3 +428,25 @@ def test_read_garbage(con, monkeypatch):
 
     with pytest.raises((FileNotFoundError, duckdb.IOException)):
         con.read_parquet("garbage_notafile")
+
+
+@pytest.mark.parametrize("i", range(5))
```

Comment:
This is so that runs of this test have a shot at running in different processes.